### PR TITLE
Homework for Week2

### PR DESCRIPTION
**Pull request by gebremedhin85**

### CODE REVIEW

**Review by Rassibassi — COMMENTED** [review]
The python script is not a .py file. So I can't see any code highlighting :(

File: week2/week2-homework [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+
+# 3. Create a class of Jet inventory with two arguments
+
+
+class Jet:
+    def __init__(self, name, country):
+        self.name = name
+        self.country = country
+        print(f'{name} {country}')
```
Comment (Rassibassi) [inline week2/week2-homework]: I understand that the print should be in a method of that class, not in the constructor

File: week2/week2-homework [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+# 6. Odd or Even
+
+
+def show_numbers(limit):
+    for num in range(0, limit + 1):
+        if num % 2 == 0:
+            print(f'{num}  EVEN')
+        else:
+            print(f'{num}  ODD')
```
Comment (Rassibassi) [inline week2/week2-homework]: Here you could do even better by separating, printing and logic. Do you know what I mean by that?

**Review by Rassibassi — COMMENTED** [review]
Good job

**Comment by gebremedhin85** [comment]
> The python script is not a .py file. So I can't see any code highlighting :(

I tried to export the .ipynb  to .py file. Should I add the bulky ipython notebook file then.

**Pull request merged**